models/question.js

```diff
@@ -0,0 +1,24 @@
+var mongoose = require('mongoose');
+
+const userSchema = mongoose.Schema({
+    question: {
+        type: String,
+        required: true
+    },
+    answer: {
```

Comment:
@matthewrapp I think we will want to look at refactoring this a bit. Currently, this will only allow for one potential answer per trivia object. Rather, I think we want to have an array of potentialAnswers that each has a text & isTrue value set inside them.

Reply:
`
answers: [{
        answer: {
                text: { type: String, required: true },
                isTrue: {type: Boolean, required: false}
        }
    }]`
I think it is fine to have a single question schema since the game will have multiple questions but here is how I would implement answers

Reply:
@ipetty14 so, Jonathan and I were talking and we were thinking that this modal only handles 1 question, not multiple question. So this modal it good for creating a question, editing a question, etc. Anything to do with the 1 question. Thoughts?